firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionViewController.swift

```diff
@@ -191,20 +190,27 @@ class TrackingProtectionViewController: UIViewController,
     // MARK: Redux
     func newState(state: TrackingProtectionState) {
         trackingProtectionState = state
-        if trackingProtectionState.showTrackingProtectionSettings {
-            enhancedTrackingProtectionMenuDelegate?.settingsOpenPage(settings: .contentBlocker)
-        } else if trackingProtectionState.showDetails {
-            showTrackersDetailsController()
-        } else if trackingProtectionState.showBlockedTrackers {
-            showBlockedTrackersController()
-        } else if trackingProtectionState.showsClearCookiesAlert {
-            onTapClearCookiesAndSiteData()
-        } else if trackingProtectionState.shouldClearCookies {
+        if let navigateTo = state.navigateTo {
+            // TODO: FXIOS-10657 connect the ETP navigation with the BVC general navigation
+        }
```

Comment:
I believe we will need to remove this since it will result in a compiler warning and we have warning set to be treated as errors.